Firebase/InstanceID/FIRInstanceIDURLQueryItem.h

```diff
@@ -18,7 +18,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-// Stand-in for NSURLQueryItem, which is only available on iOS 8.0 and up.
 @interface FIRInstanceIDURLQueryItem : NSObject
```

Comment:
Could the clients use NSURLQueryItem instead and delete the whole class?

Reply:
Good idea! Didn't realize that.